bindings/c/test/mako/mako.c

```diff
@@ -1019,6 +1112,8 @@ int init_args(mako_args_t *args) {
   args->knobs[0] = '\0';
   args->trace = 0;
   args->tracepath[0] = '\0';
+  args->traceformat = 1; /* default to JSON */
```

Comment:
My initial thought on this is that we should default to XML so that there's consistency within the tools committed to the repository. I realize that the only people using this tool right now want JSON, though, so that's maybe a little annoying. The main thing I'd want to avoid is for people to have to look up the default for every tool they use or turning this argument into a de facto required argument everywhere (because people aren't sure what they'll get).

Reply:
That's a valid argument.  Since the client trace defaults to XML today, let me follow that.